3-enrich/scala-common-enrich/.scalafmt.conf

```diff
@@ -0,0 +1,14 @@
+style = default
+align = some
+maxColumn = 140
```

Comment:
can we reduce this so that it at least fits github number of columns (I believe it's 120) or even lower? that will ease reviews

Reply:
@BenFradet I am not sure what the GH recommended max width is. 120 sounds good to me, although it will cause some reformatting (e.g. 3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/adapters/registry/UrbanAirshipAdapter.scala L53)